policy/DecamMapper.paf

```diff
@@ -421,57 +385,25 @@ datasets: {
         tables:        raw
         tables:        raw_visit
     }
+    # dataset defined in obs_base modified in this mapper as follows:
     deepCoadd_forced_metadata: {
         template:      "deepCoadd-results/forcedPhotCoadd_metadata/%(filter)s/%(tract)d/%(patch)s.boost"
-        python:        "lsst.daf.base.PropertySet"
-        persistable:   "PropertySet"
-        storage:       "BoostStorage"
     }
+    # dataset defined in obs_base modified in this mapper as follows:
     deepCoadd_forced_src: {
         template:      "deepCoadd-results/%(filter)s/%(tract)d/%(patch)s/forcedSrc-%(filter)s-%(tract)d-%(patch)s.fits"
-        python:        "lsst.afw.table.SourceCatalog"
-        persistable:   "ignored"
-        storage:       "FitsCatalogStorage"
     }
+    # dataset defined in obs_base modified in this mapper as follows:
```

Comment:
There's no reason for any one camera to be different from any other on `python` entries like this one, but I don't actually know whether this one is correct or incorrect.  Given that there has been more difference imaging work on obs_decam than other cameras recently, I suspect it's more likely this value is correct and obs_base is incorrect.  @SimonKrughoff, @ctslater, can you confirm or deny that this is the right config entry, or do you know who could?